objgraph.py

```diff
@@ -1073,6 +1091,15 @@ def _obj_node_id(obj):
     return ('o%d' % id(obj)).replace('-', '_')
 
 
+def _obj_attrs(obj, extra_node_attrs):
+    if extra_node_attrs is not None:
+        attrs = extra_node_attrs(obj)
+        return ", " + ", ".join('%s="%s"' % (name, _quote(value))
+                                for name, value in sorted(iteritems(attrs)))
```

Comment:
I wonder if it wouldn't be a good idea to add an `if value is not None` check to the generator expression, so you could do things like

```python
    extra_node_attrs=lambda obj: dict(
        URL=str(id(obj)),
        shape='diamond' if some_condition(obj) else None,
    ),
```

to conditionally add a node attribute.

Reply:
Not very useful IMHO, but I don't mind adding it.